docs/REST API/rest-api.md

```diff
@@ -9238,6 +9238,12 @@ Returns information about the specified software. By default, `versions` are sor
     "id": 12,
     "name": "Falcon.app",
     "bundle_identifier": "crowdstrike.falcon.Agent",
+    "available_apps": {
```

Comment:
```suggestion
    "available_software": {
```

I think let's go with `software` here. That way, we're consistent with the UI / guides. "Software" is the word we use to describe all types of software in Fleet: App Store (VPP) apps, Fleet-maintained apps, and custom packages.